### PR TITLE
ModAPISpriteCharacter (issue #1)

### PR DESCRIPTION
**Pull request by killaTW**
should close issue #1

### CODE REVIEW

File: src/game/client/components/items.cpp [hunk 1]
```diff
@@ -365,6 +369,62 @@ void CItems::RenderModAPISprite(const CNetObj_ModAPI_Sprite *pPrev, const CNetOb
...
+
+	RenderTools()->SelectModAPISprite(pSprite);
+	
+	Graphics()->QuadsSetRotation(Angle);
+	
+	vec2 Pos = vec2(0,0);
+	if(m_pClient->m_LocalClientID != -1 && m_pClient->m_LocalClientID == pCurrent->m_ClientId)
+	{
+		Pos = m_pClient->m_LocalCharacterPos-vec2(pCurrent->m_X,pCurrent->m_Y);
+	}
```
Comment (necropotame) [inline src/game/client/components/items.cpp]: It must be a plus, not a minus, no ?

File: src/game/client/components/items.h [hunk 2]
```diff
@@ -11,6 +11,7 @@ class CItems : public CComponent
 	void RenderFlag(const CNetObj_Flag *pPrev, const CNetObj_Flag *pCurrent, const CNetObj_GameDataFlag *pPrevGameDataFlag, const CNetObj_GameDataFlag *pCurGameDataFlag);
 	void RenderLaser(const struct CNetObj_Laser *pCurrent);
 	void RenderModAPISprite(const CNetObj_ModAPI_Sprite *pPrev, const struct CNetObj_ModAPI_Sprite *pCurrent);
+	void RenderModAPISpriteCharacter(const struct CNetObj_ModAPI_SpriteCharacter *pCurrent);
 	void RenderModAPILine(const struct CNetObj_ModAPI_Line *pCurrent);
```
Comment (necropotame) [inline src/game/client/components/items.h]: Add pPrev and mix the relative position.

**Comment by Henningstone** [comment]
@necropotame is this ok to be merged now? He updated it :)

**Comment by necropotame** [comment]
There is still a minus for the Y part. I know that the orientation of the Y axis is counter intuitive in Teeworlds, but that's the convention :-/

**Comment by Henningstone** [comment]
Where exactly? The former minus in Line 419 (before his update 416) is a plus now, where else?

**Comment by necropotame** [comment]
Line 415, Y component of vec2

**Comment by killaTW** [comment]
changed it

**Comment by necropotame** [comment]
Thanks a lot!

**Pull request merged**